### DISCUSSION TITLE
Particles like a Globe how to

### DISCUSSION THREAD

**Discussion by kwstasna** [matteobruni/tsparticles#4834] [Q&A]
Hello and thanks a lot for this awesome lib!

I would like to make something like this one or similar 😬

![2022-11-26_15-11-01 (1)](https://user-images.githubusercontent.com/13165044/204090684-c33a305c-fea3-43e6-a24e-ab540b004810.gif)

Does anyone have any experience with this type of particles design?

**Comment by matteobruni** [comment]
That is a 3D effect that cannot be done with 2D particles

**Reply by kwstasna** [reply]
What about a 2D tho? is this something doable?

**Reply by matteobruni** [reply]
Do you mean particles in a circular area?

**Reply by kwstasna** [reply]
Yes exactly, like the gif I attached but not like 3D.. and particles not to get out of the circle!

Is there by any chance this achievable?

**Reply by matteobruni** [reply]
[This config](https://github.com/matteobruni/tsparticles/blob/main/utils/configs/src/localPolygonMask.json) creates a triangle using SVG. You can replace the path with a circular one and you should be ok. It's maybe the easier way to achieve it, otherwise you can create a custom mover, plugin or path plugin. I think one of them should work for what you're trying to achieve. It's not something you can create with the standard plugins, the only one is the SVG solution, but it's mainly used inline like [this sample](https://codepen.io/matteobruni/pen/yLNpgKW) and not as an area.
